webpack.config.js

```diff
@@ -1,7 +1,17 @@
 const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
 const path = require( 'path' );
+const { spawn } = require( 'child_process' );
 // const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 
+class UAGBRunAdditionalProcess {
+	apply( compiler ) {
+		compiler.hooks.afterEmit.tapAsync( 'UAGBRunAdditionalProcess', ( compilation, callback ) => {
+			spawn( 'npm-run-all', ['--sequential', 'build:sass', 'build:placeholder'], { stdio: 'inherit' } );
```

Comment:
```suggestion
			spawn( /^win/.test(process.platform) ? 'npm-run-all.cmd' : 'npm-run-all', ['--sequential', 'build:sass', 'build:placeholder'], { stdio: 'inherit' } );
```

Tried this works for me on windows.

Reply:
This works for me. Thanks @adi3890